independent-publisher-connectors/DPIRD Science/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1223 @@
+          },
+          {
+            "$ref": "#/parameters/longitudeInQuery"
+          },
+          {
+            "$ref": "#/parameters/summerStartDateInQuery"
+          },
+          {
+            "$ref": "#/parameters/growingSeasonStartDateInQuery"
+          },
+          {
+            "$ref": "#/parameters/growingSeasonEndDateInQuery"
+          },
+          {
+            "$ref": "#/parameters/forecastDateInQuery"
+          },
+          {
+            "$ref": "#/parameters/waterUseEfficiencyInQuery"
+          },
+          {
+            "$ref": "#/parameters/evaporationInQuery"
+          }
+        ],
+        "summary": "Get Potential Yield",
+        "description": "Get Potential Yield",
+        "operationId": "GetPotentialYield"
+      }
+    },
+    "/soilwater": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "metadata": {
+                  "$ref": "#/definitions/apiMetaDataModel"
+                },
+                "data": {
+                  "$ref": "#/definitions/soilWaterModel"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Soil Water",
+        "operationId": "GetSoilWater",
+        "parameters": [
+          {
+            "$ref": "#/parameters/stationCodeLookupInQuery"
+          },
+          {
+            "$ref": "#/parameters/latitudeInQuery"
+          },
+          {
+            "$ref": "#/parameters/longitudeInQuery"
+          },
+          {
+            "name": "startDate",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The start date for the period.",
+            "x-ms-summary": "Start date"
+          },
+          {
+            "name": "endDate",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The end date for the period.",
+            "x-ms-summary": "End date"
+          },
+          {
+            "name": "soilType",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "A code for a type of soil",
+            "x-ms-summary": "Soil Code",
+            "enum": [
+              "gravel",
+              "shallow-soil",
+              "sand",
+              "sandy-earth",
+              "shallow-sandy-duplex",
+              "deep-sandy-duplex",
+              "shallow-loamy-duplex",
+              "deep-loamy-duplex",
+              "loamy-earth",
+              "clay"
+            ]
+          },
+          {
+            "name": "faoInitialisationDays",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Number of days in crop initialisation period, used in estimation of crop evapotranspiration.",

```

Comment:
All descriptions, summaries, titles, etc. must be free from grammar and spelling errors. Please review them.

Reply:
Line 396 is correct as the source API documentation at DPIRD: 

![image](https://user-images.githubusercontent.com/4979626/162562586-da5e05ec-3e5a-450a-9347-9186aee470e1.png)

Reply:
Nope, that one is also correct too. What I will do is run through the spell checker but flag if this is as per the original API. Most of these descriptions are cut/paste from source so a lot of the errors are at source.

Reply:
Fixed as suggested in various spots